next-cloudinary/src/components/CldImage/CldImage.js

```diff
@@ -56,6 +56,11 @@ const CldImage = props => {
     }
   }
 
+  if (props.src && props.useUrlTransformations) {
```

Comment:
i know `use*` was in the original issue, but i may be more in favor of `preserveTransformations`

thinking through it, `use` is more hook oriented in React apps and `useUrlTrasnformations` could be ambiguous, making the assumption someone knows we're referring to the original URL transformations